src/config/database/migrations/20200802012857-create-access-tokens.js

```diff
@@ -0,0 +1,43 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('access_tokens', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+      },
+      user_id: {
+        type: Sequelize.INTEGER,
+        references: { model: 'users', key: 'id' },
+        onUpdate: 'CASCADE',
+        onDelete: 'CASCADE',
+        allowNull: false,
+      },
+      token: {
+        type: Sequelize.TEXT,
+        allowNull: false,
+      },
+      refresh_token: {
```

Comment:
Codacy found an issue: [Identifier 'refresh_token' is not in camel case.](https://app.codacy.com/manual/lucas.assuncao.p/node-bloodboiler-sequelized/pullRequest?prid=5950393)